### ISSUE TITLE
Connect the deleted database by default in the created "query-2.psql" file after executing "Create Function Query..." from command palette

### ISSUE THREAD

**Issue by v-xinda** [microsoft/vscode-cosmosdb#2097]
**OS:** All
**Build Version:** 20230205.1

**Repro Steps:**

1. Create a PostgreSQL account -> Create a database "Data01".
2. Delete the database "Data01".
3. F1 -> Click "Create Function Query..." -> Provide function name -> Select return type -> Enter.
4. Check the created `query-2.psql` file.

**Expect:**
Connect an existing database or show "connect to database" (Like `Scrapbook.mongo` file) in the created `query-2.psql` file.
<img width="570" alt="Screenshot 2023-02-08 at 17 00 36" src="https://user-images.githubusercontent.com/57662131/217487220-982cc31d-de76-4ecd-98b4-d1c26acde88e.png">

**Actual:**
1. Connect the deleted database "Data01" in the created `query-2.psql` file.
<img width="753" alt="Screenshot 2023-02-08 at 17 00 09" src="https://user-images.githubusercontent.com/57662131/217486588-d5bc5a93-6f02-42f3-84f2-5ee37779de4d.png">
2. An error occurs after clicking "Execute Query" in the created `query-2.psql` file.
<img width="640" alt="Screenshot 2023-02-08 at 16 56 35" src="https://user-images.githubusercontent.com/57662131/217486936-4ac69bf8-d829-47d4-8cef-0c7683a4a4fb.png">

**More Info:**
This issue also reproduces for the stored procedures query.

**Comment by v-xinda** [comment]
@alexweininger  No, it's not a regression. This issue also reproduces on the current release build.

**Comment by JasonYeMSFT** [comment]
This is because deleting a postgres database requires first connecting to it, which will leave the deleted database in the cached connection data. A simple mitigation would be to always clear the connection after deleting a postgres database since we know it will always invalidate the current connection.
